package.json

```diff
@@ -20,6 +20,7 @@
   },
   "homepage": "http://cookieconsent.insites.com/",
   "scripts": {
+    "build": "node_modules/.bin/gulp build",
```

Comment:
this change is not necessary anymore because we updated the package to contain build scripts which automagically use the node_modules version of gulp rather than the global version.